src/test/java/net/snowflake/client/jdbc/DatabaseMetaDataLatestIT.java

```diff
@@ -1557,44 +1614,47 @@ public void testGetStreams() throws SQLException {
       final String targetTable = "T0";
       String tableName = database + "." + schema + "." + targetTable;
 
-      Statement statement = con.createStatement();
-      statement.execute("create or replace table " + targetTable + "(C1 int)");
-      statement.execute("create or replace stream " + targetStream + " on table " + targetTable);
-
-      DatabaseMetaData metaData = con.getMetaData();
-
-      // match stream
-      ResultSet resultSet =
-          metaData.unwrap(SnowflakeDatabaseMetaData.class).getStreams(database, schema, "%");
-      verifyResultSetMetaDataColumns(resultSet, DBMetadataResultSetMetadata.GET_STREAMS);
-      Set<String> streams = new HashSet<>();
-      while (resultSet.next()) {
-        streams.add(resultSet.getString(1));
+      try (Statement statement = con.createStatement()) {
```

Comment:
You could also just create this statement in the same try with the connection above.

Reply:
Fixed.